status/src/server.rs

```diff
@@ -22,14 +23,10 @@ pub struct Server {
 
 impl Server {
     pub async fn from_env() -> Result<Self> {
-        let socket_addr = dotenv::var("API_SOCKET_ADDR").and_then(|v| {
```

Comment:
Still need that dotenv error in the errors module?

Reply:
ah no we do not.. good catch.

Reply:
hmm not quite.. `dotenv::dotenv()` still needs the Error represented in the crates

Reply:
> per the docs, what does "load the .env file" mean? is that like sourcing the file into the runtime environment from within the application in case your terminal session or systemd unit or whatever doesn't do so or is it specific to the dotenv crate such that we don't need that function anymore either?

true.. though it's useful on the current server(s) since the env variables are _not_ set outside of the `.env` file which this function loads into the environment